src/canvas.tsx

```diff
@@ -58,7 +58,7 @@ export const Canvas = React.memo(
     const [bind, size] = useMeasure()
     const [intersects, setIntersects] = useState([])
     const [raycaster] = useState(() => new THREE.Raycaster())
-    const [mouse] = useState(() => new THREE.Vector2())
+    const [mouse] = useState(() => new THREE.Vector3())
```

Comment:
oh my, what did i do there ... good catch! 😅

Reply:
The promise of TypeScript is real!